extensions/wikia/Chat2/Chat.i18n.php

```diff
@@ -119,7 +119,9 @@
 	'chat-browser-is-notsupported' => '	Your browser is not supported. For the best experience, use a <a href="http://community.wikia.com/wiki/Help:Supported_browsers">newer browser</a>.',
 	'chat-checkuser-join-action' => 'Joined Chat',
 
-	'chat-status-away' => 'Away'
+	'chat-status-away' => 'Away',
+	
+	'chat-limit-reached' => 'Sorry, but your message is too long. You may only send no more than $1 characters',
```

Comment:
Please, add a message documentation under the [qqq key](https://github.com/Wikia/app/blob/f8b5b3d50370428a7830d71835cc127a0d784e60/extensions/wikia/Chat2/Chat.i18n.php#L138-L206)